libr/bin/bin.c

```diff
@@ -383,39 +386,38 @@ R_API bool r_bin_open_io(RBin *bin, RBinOptions *opt) {
 		// change the name to something like
 		// <xtr_name>:<bin_type_name>
 		r_list_foreach (bin->binxtrs, it, xtr) {
-			if (xtr->check_bytes (buf_bytes, opt->sz)) {
+			if (!xtr->check_buffer) {
+				eprintf ("Missing check_buffer callback for '%s'\n", xtr->name);
```

Comment:
Use `R_LOG_ERROR`

Reply:
we should probably do this rlogification in another pr, i alos want to do an RCoreIO killification. the RIO api is good enouogh to not require wrappers in core anymore for example